builtin/providers/triton/provider.go

```diff
@@ -0,0 +1,130 @@
+package triton
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/joyent/gocommon/client"
+	"github.com/joyent/gosdc/cloudapi"
+	"github.com/joyent/gosign/auth"
+)
+
+// Provider returns a terraform.ResourceProvider.
+func Provider() terraform.ResourceProvider {
+	return &schema.Provider{
+		Schema: map[string]*schema.Schema{
+			"account": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				DefaultFunc: schema.EnvDefaultFunc("SDC_ACCOUNT", ""),
+			},
+
+			"url": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				DefaultFunc: schema.EnvDefaultFunc("SDC_URL", "https://us-west-1.api.joyentcloud.com"),
+			},
+
+			"key_path": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				DefaultFunc: schema.EnvDefaultFunc("SDC_KEY_PATH", ""),
+			},
+
+			"key_material": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				DefaultFunc: schema.EnvDefaultFunc("SDC_KEY_MATERIAL", ""),
+			},
```

Comment:
Agreed. I added this when first doing the provider and later changed it, this is a vestigal remnant from that. Looks like it can be removed with impunity. :)